docs/cli_README.md

```diff
@@ -5,21 +5,52 @@
 ### Installation:
 - Select, Download and Install latest binary from [Releases](https://github.com/fabric8-analytics/cli-tools/releases)
 
+#### curl
+
+Example:
+```
+# For Linux 64Bit, Downloading v0.0.1
+$. curl -O -L https://github.com/fabric8-analytics/cli-tools/releases/download/v0.0.1/crda_0.0.1_Linux_64bit.tar.gz
```

Comment:
Also one liner would be great to have.
e.g
```
curl -s -L https://github.com/fabric8-analytics/cli-tools/releases/download/v0.0.1/crda_0.0.1_Linux_64bit.tar.gz | tar xvz -C .
```

Reply:
Ack